packages/protocol/contracts/L1/TaikoL1.sol

```diff
@@ -230,6 +234,9 @@ contract TaikoL1 is EssentialContract, ITaikoL1, TaikoEvents, TaikoErrors {
             blockMaxGasLimit: 240_000_000,
             livenessBond: 250e18, // 250 Taiko token
             blockSyncThreshold: 32,
+            // Setting this to 16 means on average, on average 8 blocks will be verified in each
+            // verifyBlocks().
+            verificationFrequencyFactor: 16,
```

Comment:
Maybe we can omit this extra config value and reuse the `maxBlocksToVerify` ?
https://github.com/taikoxyz/taiko-mono/pull/17553/files#r1635177191